grid_layout.go

```diff
@@ -0,0 +1,10 @@
+package gxui
+
+type GridLayout interface {
+	Control
+
+	Container
```

Comment:
You probably want to use [Parent](https://github.com/google/gxui/blob/master/container.go#L19) instead of [Container](https://github.com/google/gxui/blob/master/container.go#L25) as adding or removing child controls here wouldn't make sense.